packages/react-components/src/components/MeetingConferencePhoneInfo.tsx

```diff
@@ -0,0 +1,260 @@
+
+/* @conditional-compile-remove(teams-meeting-conference) */
+import {
+  themedPhoneInfoModalStyle,
+  titleClassName,
+  titleContainerClassName,
+  phoneInfoTextStyle,
+  phoneInfoIcon,
+  phoneInfoInctructionLine,
+  phoneInfoStep,
+  stepTextStyle,
+  infoConnectionLinkStyle
+} from './styles/TeamsMeetingConferenceInfo';
+/* @conditional-compile-remove(teams-meeting-conference) */
+import { _preventDismissOnEvent } from '@internal/acs-ui-common';
+/* @conditional-compile-remove(teams-meeting-conference) */
+import { useLocale } from '../localization';
+/* @conditional-compile-remove(teams-meeting-conference) */
+import { isPossiblePhoneNumber, parsePhoneNumber } from 'libphonenumber-js';
+
+/* @conditional-compile-remove(teams-meeting-conference) */
+/**
+ * @beta
+ * Information for conference phone info
+ */
+export interface ConferencePhoneInfo {
+  phoneNumber: string;
+  conferenceId: string;
+  isTollFree: boolean;
+  country?: string;
+  city?: string;
+}
+
+/* @conditional-compile-remove(teams-meeting-conference) */
+/**
+ * strings for phone info modal
+ * @beta
+ */
+export interface MeetingConferencePhoneInfoModalStrings {
+  /**
+   * Header for the phone info modal
+   */
+  meetingConferencePhoneInfoModalTitle?: string;
+  /**
+   * Phone number instruction
+   */
+  meetingConferencePhoneInfoModalDialIn?: string;
+  /**
+   * Meeting ID instruction
+   */
+  meetingConferencePhoneInfoModalMeetingId?: string;
+  /**
+   * Wait for phone connection
+   */
+  meetingConferencePhoneInfoModalWait?: string;
+  /**
+   * Tool Free Phone Label
+   */
+  meetingConferencePhoneInfoModalTollFree?: string;
+  /**
+   * Tool Phone Label
+   */
+  meetingConferencePhoneInfoModalToll?: string;
+  /**
+   * No phone number available message
+   */
+  meetingConferencePhoneInfoModalNoPhoneAvailable?: string;
+}
+
+/* @conditional-compile-remove(teams-meeting-conference) */
+/**
+ * @beta
+ * MeetingConferencePhoneInfoModal Component Props.
+ */
+export interface MeetingConferencePhoneInfoModalProps {
+  conferencePhoneInfoList: ConferencePhoneInfo[];
+  showModal?: boolean;
+  onDismissMeetingPhoneInfoSettings?: () => void;
+}
+
+/* @conditional-compile-remove(teams-meeting-conference) */
+/**
+ * @beta
+ * a component for setting spoken languages
+ */
+export const MeetingConferencePhoneInfoModal = (props: MeetingConferencePhoneInfoModalProps): JSX.Element => {
+  const { conferencePhoneInfoList, showModal, onDismissMeetingPhoneInfoSettings } = props;
+
+  const theme = useTheme();
+  const strings = useLocale().strings.MeetingConferencePhoneInfo;
+
+  const onDismiss = useCallback((): void => {
+    if (onDismissMeetingPhoneInfoSettings) {
+      onDismissMeetingPhoneInfoSettings();
+    }
+  }, [onDismissMeetingPhoneInfoSettings]);
+
+  const PhoneInfoModalStyle: Partial<IModalStyles> = useMemo(() => themedPhoneInfoModalStyle(theme), [theme]);
+
+  return (
+    <>
+      {
```

Comment:
Can these be remove? (or do they have a purpose?)

Reply:
Done